### DISCUSSION TITLE
Alpha 3 smartbar issue

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [florisboard/florisboard#2028] [General]
~~dunno y but after alpha 3 there is problem in bitwarden autofill service~~

~~previously whenever there is a log screen a bitwarden popup shows on action bar/smartbar now there is none i tried multiple times i need to open the app & copy paste manually everytime~~

~~dunno if this is app fault or bitwarden side issue but it started happening just after i updated it to alpha 03 meaning today.~~

~~any help would be appreciated~~

Stable 0.316

![Screenshot_2022-08-29-08-51-39-205_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/38470944/187119694-e4307d79-d628-4559-8571-4c7f80ef8fb8.jpg)

alpha 03

![Screenshot_2022-08-29-08-51-53-020_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/38470944/187119714-bcb5af86-9ecd-45ba-83df-5ceeeecb9296.jpg)

Edit:

problem is shouldnt auto open/collapse closes smartbar when there is another entry needed or something like bitwarden shown in screen currently it doesnt collapse.

**Comment by klaurence** [comment]
I can't reproduce this with alpha3. Which options in the `Auto-fill services` setting of your Bitwarden app are on?

**Reply by HikaruIchijyo** [reply]
yep autofill is on it sometimes does autofill but sometimes there is no bitwarden option at all

**Reply by HikaruIchijyo** [reply]
every option in autofill is on

**Reply by HikaruIchijyo** [reply]
i tried multiple times & it doesnt show

**Reply by klaurence** [reply]
Does the popup show if you use other keyboards or Florisboard Stable?

**Reply by HikaruIchijyo** [reply]
> Does the popup show if you use other keyboards or Florisboard Stable?

it was working fine as i stated i am gonna test with stable & will get back

**Reply by HikaruIchijyo** [reply]
> Does the popup show if you use other keyboards or Florisboard Stable?

Stable

![Screenshot_2022-08-29-08-51-39-205_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/38470944/187119612-db2bf030-bc0e-4315-891a-be797a77fcd8.jpg)

Alpha03

![Screenshot_2022-08-29-08-51-53-020_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/38470944/187119643-4706572e-c4c2-41d3-b9aa-fdff9a466000.jpg)

**Reply by HikaruIchijyo** [reply]
my bad the problem  is overflow menu autoopen(smart bar) which stops showing bitwarden when u close it it is working fine
